x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -54,9 +53,7 @@ const DEFAULT_SORT = {
   field: 'impact' as const,
 };
 
-function getLatencyAggregationTypeLabel(
-  latencyAggregationType?: LatencyAggregationType
-) {
+function getLatencyAggregationTypeLabel(latencyAggregationType?: string) {
```

Comment:
I'm not sure if this function should return a label for `average` if the latencyAggregationType is not set or unknown.